test/unix/main.ml

```diff
@@ -217,6 +217,7 @@ module Close = struct
     Index.replace w1 k v;
     Hashtbl.replace tbl k v;
     Index.close w1;
+
```

Comment:
Unrelated, this is ocamlformat.
Reported here : https://github.com/ocaml-ppx/ocamlformat/issues/1019

Reply:
probably best to not commit this part of that PR